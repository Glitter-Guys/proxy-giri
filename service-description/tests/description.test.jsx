import React from 'react';
import ReactDOM from 'react-dom';
import {shallow, mount} from 'enzyme';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import Description from '../client/src/components/Description';

Enzyme.configure({ adapter: new Adapter() });

var testDescription = "<p>This is the test paragraph</p>"

it('renders correctly', () => {
  const compDesc = renderer
    .create(<Description details={testDescription} />)
    .toJSON();
  expect(compDesc).toMatchSnapshot();
});

it('should add details to the page', () => {
  const wrapper = shallow(<Description details={testDescription}/>)
  const div = wrapper.find('.runningText');
  expect(div).toHaveLength(1);
  expect(div.text()).toBe("This is the test paragraph");
})