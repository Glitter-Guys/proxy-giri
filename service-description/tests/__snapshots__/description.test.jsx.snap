// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="runningText"
>
  <p>
    This is the test paragraph
  </p>
</div>
`;
